examples/ansibleee-playbook-local.yaml

```diff
@@ -28,7 +18,7 @@ spec:
     - name: ANSIBLE_ENABLE_TASK_DEBUGGER
       value: "True"
   extraMounts:
-      extraVolType: Ceph
+    - extraVolType: Ceph
```

Comment:
+1 thanks, you can just squash your commits, otherwise lgtm!

Reply:
Github automatically choose "squash changes" when merging to main.